### PR TITLE
Qt5 14 1 update

### PR DESCRIPTION
**Pull request by PeterPetrik**

### CODE REVIEW

File: android/Dockerfile [hunk 1]
```diff
@@ -56,24 +56,25 @@ RUN apt-get install -qq -y --no-install-recommends \
...
-qt.qt5.$(echo "${QT_VERSION}" | tr -d .).android_arm64_v8a,\
-qt.qt5.$(echo "${QT_VERSION}" | tr -d .).android_armv7\
-    /tmp/qt/extract-qt-installer.sh /tmp/qt/installer.run "${QT_PATH}" \
-    && find "${QT_PATH}" -mindepth 1 -maxdepth 1 ! -name "${QT_VERSION}" -exec echo 'Cleaning Qt SDK: {}' \; -exec rm -r '{}' \; \
-    && rm -rf /tmp/qt
+RUN mkdir -p /opt/Qt/${QT_VERSION}
+RUN mkdir -p /tmp/qt
+RUN curl -k -Lo /tmp/qt/qt-${QT_VERSION}.tar.gz "https://www.dropbox.com/s/76h23kjqt3h92g2/android.tar.gz?dl=0"
+RUN cd /opt/Qt/${QT_VERSION} && tar -xvzf /tmp/qt/qt-${QT_VERSION}.tar.gz
+RUN rm -rf /tmp/qt/qt-${QT_VERSION}.tar.gz
```
Comment (PeterPetrik) [inline android/Dockerfile:77]: @saberraz this should be merged to one RUN command with `&&`

File: android/Dockerfile [hunk 2]
```diff
@@ -100,10 +101,19 @@ RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales
 
 ####################################################################
 # now OSGeo libs
+# TODO move above!
+
```
Comment (PeterPetrik) [inline android/Dockerfile:105]: remove @saberraz

File: android/Dockerfile [hunk 3]
```diff
@@ -100,10 +101,19 @@ RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales
...
 RUN /usr/src/android/distribute.sh -m gdal
-RUN /usr/src/android/distribute.sh -m qgis && mv /usr/src/build-android/stage /home/input-sdk && rm -rf /usr/src
+RUN /usr/src/android/distribute.sh -m protobuf
+RUN /usr/src/android/distribute.sh -m qca
+RUN /usr/src/android/distribute.sh -m qtkeychain
+RUN /usr/src/android/distribute.sh -m libspatialite
+RUN /usr/src/android/distribute.sh -m qgis
+
+RUN mv /usr/src/build-android/stage /home/input-sdk && rm -rf /usr/src
```
Comment (PeterPetrik) [inline android/Dockerfile:119]: this should be removed and used the line as it was before @saberraz

File: android/Dockerfile [hunk 4]
```diff
@@ -100,10 +101,19 @@ RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales
...
 # now OSGeo libs
+# TODO move above!
+
 
 COPY config_docker.conf /usr/src/android/config.conf
 COPY tools /usr/src/android/tools
 COPY recipes /usr/src/android/recipes
 COPY distribute.sh /usr/src/android/distribute.sh
+
 RUN /usr/src/android/distribute.sh -m gdal
```
Comment (PeterPetrik) [inline android/Dockerfile:112]: we should remove this gdal line too

**Comment by saberraz** [comment]
The latest changes reduced the size of the image. Looks good now!

**Pull request merged**